andela/balder-ah-frontend
# #163221779 Users should be able to create and view articles

#### What does this PR do?
Adds functionality for users to be able to create and view articles


#### Description of Task to be completed?
* Creates the frontend for users to be able to create new articles
* Creates the frontend for users to be able to read/view an article
 

#### How should this be manually tested?
* Clone the repo and `cd` into the folder
* Run `npm install` to install all the dependencies
* Run `npm run start:dev` to start the server
* In your browser visit localhost:3000/create-article (you will be redirected to login if you aren't already)
* after creating the article, it should redirect you to the article that was just created



#### Any background context you want to provide?

N/A

#### What are the relevant pivotal tracker stories? (If applicable)

[#163221779](https://www.pivotaltracker.com/story/show/163221779)

#### Screenshots (If applicable)
<img width="1440" alt="screen shot 2019-01-20 at 10 50 48 pm" src="https://user-images.githubusercontent.com/29441898/51445495-e5442080-1d05-11e9-81de-e5b300b2b1d1.png">

<img width="1440" alt="screen shot 2019-01-22 at 10 41 45 am" src="https://user-images.githubusercontent.com/29441898/51526518-d7d88480-1e32-11e9-95d8-f52f0e4616f1.png">


<img width="417" alt="screen shot 2019-01-22 at 10 42 00 am" src="https://user-images.githubusercontent.com/29441898/51526525-de66fc00-1e32-11e9-9b42-41a09493b606.png">


<img width="417" alt="screen shot 2019-01-20 at 10 20 21 pm" src="https://user-images.githubusercontent.com/29441898/51445410-b9746b00-1d04-11e9-82e5-3716f4d50027.png">





#### Questions:
N/A
